js/src/Ice/Endpoint.js

```diff
@@ -1,62 +1,29 @@
-//
-// Copyright (c) ZeroC, Inc. All rights reserved.
-//
-
-import { Ice as Ice_Version } from "./Version.js";
-const { EncodingVersion } = Ice_Version;
+// Copyright (c) ZeroC, Inc.
 
 /**
  *  Base class providing access to the endpoint details.
  **/
-export class EndpointInfo {
-    constructor(underlying = null, timeout = 0, compress = false) {
```

Comment:
Application code should not construct EndpointInfo/ConnectionInfo, so there is no reason to provide constructors in TS.